vector/src/main/java/im/vector/view/HomeSectionView.java

```diff
@@ -51,16 +51,16 @@
     private static final String LOG_TAG = HomeSectionView.class.getSimpleName();
 
     @BindView(R.id.section_header)
-    TextView mHeader;
+    public TextView mHeader;
```

Comment:
Should these be exposed with getters, rather than directly, so calling code can't change the object reference?

Reply:
I could do that. However, no one uses that anymore. In fact, kotlin's default access type is public.

Reply:
Wow so they don't care about encapsulation!! so are you only adding public here to make the access explicit and more obvious?

Reply:
the way it had been coded is accessible from the same package (without using get or set methods). Since I have created a separate package outside for the new home feature, I had to make them public.

Reply:
?? I thought you said the default access type in kotlin is public.  As there is no access type defined would they not already be public?

Reply:
that particular class is in Java. The classes I am writing are in Kotlin.

Reply:
Ah ok, I don't like the pattern of breaking encapsulation by exposing class variables outside of the class (and sometimes sub-classes is ok), I guess for now this can be acceptable.